Project.toml

```diff
@@ -20,6 +20,6 @@ Antic_jll = "e21ec000-9f72-519e-ba6d-10061e575a27"
 AbstractAlgebra = "^0.9.1"
 BinaryProvider = "0.4, 0.5"
 julia = "1"
-LoadFlint = "^0.2.1"
-Arb_jll = "= 2.17.0"
-Antic_jll = "^0.1.0"
+LoadFlint = "= 0.3.0"
+Arb_jll = "= 2.18.0"
+Antic_jll = "= 0.2.1"
```

Comment:
Could you please explain what the rationale is for fixing the precise version here? In particular for the JLLs? Perhaps the reason could be inserted as a comment into the Project.toml?

Reply:
We probably don't need it on LoadFlint, but the other packages don't seem to following SemVer. So `FLINT_jll` version 2.7.0 or even 2.6.1 could in theory break Nemo that worked with version 2.6.0. So there is no safe upper bound that we could use.